<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="vincent-asus" name="org.vas.test.TestRest" tests="2" failures="1" timestamp="5 Apr 2015 14:52:13 GMT" time="0.970" errors="0">
  <testcase name="itShouldQueryGoogle" time="0.485" classname="org.vas.test.TestRest"/>
  <testcase name="itShouldAuthenticateTestUser" time="0.485" classname="org.vas.test.TestRest">
    <failure type="java.lang.RuntimeException" message="org.codehaus.jackson.JsonParseException: Unexpected character (&#039;&lt;&#039; (code 60)): expected a valid value (number, String, array, object, &#039;true&#039;, &#039;false&#039; or &#039;null&#039;)
 at [Source: java.io.StringReader@31bcf236; line: 1, column: 2]">
      <![CDATA[java.lang.RuntimeException: org.codehaus.jackson.JsonParseException: Unexpected character ('<' (code 60)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
 at [Source: java.io.StringReader@31bcf236; line: 1, column: 2]
	at org.vas.test.rest.RestImpl.get(RestImpl.java:65)
	at org.vas.test.TestRest.itShouldAuthenticateTestUser(TestRest.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.codehaus.jackson.JsonParseException: Unexpected character ('<' (code 60)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
 at [Source: java.io.StringReader@31bcf236; line: 1, column: 2]
	at org.codehaus.jackson.JsonParser._constructError(JsonParser.java:1432)
	at org.codehaus.jackson.impl.JsonParserMinimalBase._reportError(JsonParserMinimalBase.java:521)
	at org.codehaus.jackson.impl.JsonParserMinimalBase._reportUnexpectedChar(JsonParserMinimalBase.java:442)
	at org.codehaus.jackson.impl.ReaderBasedParser._handleUnexpectedValue(ReaderBasedParser.java:1198)
	at org.codehaus.jackson.impl.ReaderBasedParser.nextToken(ReaderBasedParser.java:485)
	at org.codehaus.jackson.map.ObjectMapper._initForReading(ObjectMapper.java:2770)
	at org.codehaus.jackson.map.ObjectMapper._readMapAndClose(ObjectMapper.java:2718)
	at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1863)
	at org.vas.test.rest.RestImpl.body(RestImpl.java:145)
	at org.vas.test.rest.RestImpl.extractResponse(RestImpl.java:195)
	at org.vas.test.rest.RestImpl.get(RestImpl.java:63)
	... 25 more
]]>
    </failure>
  </testcase> <!-- itShouldAuthenticateTestUser -->
</testsuite> <!-- org.vas.test.TestRest -->
